src/client/components/widgets/GoSearchInput/index.tsx

```diff
@@ -125,12 +125,13 @@ const GoSearchInput: FunctionComponent<GoSearchInputProps> = ({
           value={query}
           onChange={(e) => onQueryChange(e.target.value)}
           onKeyPress={onKeyPress}
+          variant="outlined"
           InputProps={{
             className: classes.searchInput,
-            disableUnderline: true,
+            // disableUnderline: true,
```

Comment:
do we plan to eventually restore this config option?

Reply:
nope, will remove